Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5334,34 +5653,137 @@ def panorama_install_latest_content_update(target: str):
     return result
 
 
-def panorama_install_latest_content_update_command(target: Optional[str] = None):
+def panorama_install_latest_content_update_command(args):
     """
         Check jobID of content content install status
     """
     if DEVICE_GROUP:
         raise Exception('Content download status is only supported on Firewall (not Panorama).')
-    result = panorama_install_latest_content_update(target)
+    
+    target = args.get('target')
+    use_polling = args.get('polling', 'false') == 'true'
+    job_id = args.get('job_id', None)
+    timeout = int(args.get('timeout', 600))
+    interval_in_seconds = int(args.get('interval_in_seconds', 60))
+    cmd = demisto.command()
+    script_results = []
+
+    # Is polling requested
+    if use_polling:
+        ScheduledCommand.raise_error_if_not_supported()
+
+        # Create new job if this is not an existing polling job
+        if not job_id:
+
+            # Create a new installation schedule
+            result = panorama_install_latest_content_update(target)
+            if result['response']['@status'] == 'error':
+                if 'msg' in result['response'] and 'line' in result['response']['msg']:
+                    message = '. Reason is: ' + result['response']['msg']['line']
+                    raise Exception('Update content failed' + message)
+                else:
+                    raise Exception('Update content failed.')
 
-    if 'result' in result['response']:
-        # installation has been given a jobid
-        content_install_info = {
-            'JobID': result['response']['result']['job'],
-            'Status': 'Pending'
-        }
-        entry_context = {"Panorama.Content.Install(val.JobID == obj.JobID)": content_install_info}
-        human_readable = tableToMarkdown('Result:', content_install_info, ['JobID', 'Status'], removeNull=True)
+            if 'response' not in result or 'result' not in result['response'] or 'job' not in result['response']['result']:
+                raise Exception('Missing JobID in response.')
```

Comment:
can we log the response itself here?

Reply:
Logging the response to the user would end up with the polling command filling the warroom with data. It's neater and nicer for the user experience to see a single "polling" command (with the timer) rather than multiple messages of the same command.